app/controllers/verify/finance_controller.rb

```diff
@@ -25,6 +32,30 @@ def render_form
       end
     end
 
+    def confirm_finance_with_vendor?
+      session_id = idv_session.resolution.session_id
+      idv_session.financials_confirmation = idv_agent.submit_financials(financials, session_id)
+      track_idv_event
+      idv_session.financials_confirmation.success?
+    end
+
+    def financials
+      finance_type = idv_finance_form.finance_type
+      { finance_type => idv_finance_form.idv_params[finance_type] }
+    end
+
+    def financials_confirmation_failed?
+      idv_finance_form.finance_type && !idv_session.financials_confirmation.try(:success?)
+    end
```

Comment:
this is not tested because it is a placeholder for when the frontend work is done by @hursey013